tests/test_bulletproof.py

```
@@ -3,55 +3,56 @@
+        false_vals = ["no", "false", "f", "n", "0", "NO", "FALSE", "False", "N"]
         for val in false_vals:
             assert str_to_bool(val) is False
-        
+
         # Test boolean passthrough
         assert str_to_bool(True) is True
         assert str_to_bool(False) is False
-        
+
         # Test invalid values
         with pytest.raises(Exception):
```

Ruff (0.13.1)

Do not assert blind exception: `Exception`

(B017)